app-editors/ghostwriter/ghostwriter-1.7.4-r1.ebuild

```diff
@@ -21,6 +21,7 @@ RDEPEND="
 	dev-qt/qtprintsupport:5
 	dev-qt/qtwebkit:5
 	dev-qt/qtwidgets:5
+	dev-qt/qtsvg:5
```

Comment:
please keep lexicographical sorting order, sorry I missed that earlier ;)

Reply:
Thanks. Sorry for my little mistakes.